hr_holidays_legal_leave/models/hr_holidays_status.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models, api
+
+
+class HolidaysType(models.Model):
+    _inherit = "hr.holidays.status"
+
+    is_annual = fields.Boolean(
+        'Legal/Annual',
+        compute='_compute_is_annual',
+        inverse='_inverse_is_annual',
+        help='Use this Leave type as Legal/Annual for current company. '
+             'One and only one leave type can have this checkbox. '
+             'You cannot unset it directly. '
+             'Set it on another Leave type instead. '
+    )
+
+    @api.multi
+    def _compute_is_annual(self):
+        company = self.env.user.company_id
+        for r in self:
+            r.is_annual = r.id == company.legal_holidays_status_id.id
```

Comment:
It doesn't work with multiple records:
```
  File "/mnt/files/src/odoo-10/odoo/fields.py", line 916, in _compute_value
    getattr(records, self.compute)()
  File "/mnt/files/git/oca_hr-10/hr_holidays_legal_leave/models/hr_holidays_status.py", line 23, in _compute_is_annual
    .is_annual = True
  File "/mnt/files/src/odoo-10/odoo/fields.py", line 875, in __set__
    record.ensure_one()
  File "/mnt/files/src/odoo-10/odoo/models.py", line 4783, in ensure_one
    raise ValueError("Expected singleton: %s" % self)
ValueError: Expected singleton: hr.holidays.status()
```

``write`` make the trick, but as I understand in computed fields ``write`` is not the same as ``__set__``

        self.filtered(lambda x: x == company.legal_holidays_status_id)\
            .write({'is_annual': True})